Tests/TweenTests.cs

```diff
@@ -91,6 +91,8 @@ public class TweenTests : MonoBehaviour {
         this.target.TweenSpriteRendererColor (Color.white, 1).SetFrom (Color.red);
       if (GUILayout.Button ("TextMeshAlpha"))
         this.target.TweenTextMeshAlpha (1, 1).SetFrom (0);
+      if (GUILayout.Button ("VolumeWeight"))
+        this.target.TweenVolumeWeight (1, 1).SetFrom (0);

```

Comment:
I haven't tested it yet, but I think this could error when the assembly definition `SRP_AVAILABLE` is not defined.